turborepo-tests/helpers/setup_example_test.sh

```diff
@@ -21,13 +21,16 @@ fi
 
 echo "node --version: $(node --version)"
 
+temp_dir=$(mktemp -d)
+echo "Created temporary directory: $temp_dir"
+
 # Convert to the right package manager
 if [ "$package_manager" == "npm" ]; then
-  package_manager_command="node ../../../packages/turbo-workspaces/dist/cli.js convert . npm --ignore-unchanged-package-manager"
+  package_manager_command="npx @turbo/workspaces convert . npm --ignore-unchanged-package-manager"
```

Comment:
No, future reviewer, I do not enjoy this code. However, I'm not sure how else I'd want to do it now that I'm working in a temp dir. 🤔